packages/lesswrong/components/common/SearchBar.jsx

```diff
@@ -1,23 +1,56 @@
 import React, { Component } from 'react';
 import { registerComponent, Components, getSetting } from 'meteor/vulcan:core';
 import PropTypes from 'prop-types';
-import {
-  InstantSearch,
-  Hits,
-  SearchBox,
-  Pagination,
-  Configure,
-  Index } from 'react-instantsearch/dom';
-import FontIcon from 'material-ui/FontIcon';
-
-const closeIconStyle = {
-  fontSize: '14px',
-}
-
-const searchIconStyle = {
-  position: 'absolute',
-  padding: '12px',
-}
+import { InstantSearch, SearchBox } from 'react-instantsearch/dom';
+import { withStyles } from '@material-ui/core/styles';
+import classNames from 'classnames';
+import Icon from '@material-ui/core/Icon'
+
+const styles = theme => ({
+  searchBar: {
+    display: "inline-block",
+    position: "relative",
+    "& .ais-SearchBox__submit":{
```

Comment:
In general some comments on this CSS might be helpful.

Reply:
Yeah, I'll add that in. Basically our search components come with weird classes we don't have much direct control over, but which we can change via CSS.

Reply:
(this isn't meant to be "a way we do things", so much as a particular hack that was worth it in this case)